firmware/config/boards/core48/board_configuration.cpp

```diff
@@ -0,0 +1,194 @@
+
+	// we only have pwm/dir, no dira/dirb
+	engineConfiguration->etb_use_two_wires = false;
+}
+
+static void 
+setupVbatt() {
+	// 5.6k high side/10k low side = 1.56 ratio divider
+	engineConfiguration->analogInputDividerCoefficient = 1.56f;
+	
+	// 6.34k high side/ 1k low side
+	engineConfiguration->vbattDividerCoeff = (6.34 + 1) / 1; 
+
+	// Battery sense on PA7
+	engineConfiguration->vbattAdcChannel = EFI_ADC_9;
+
+	engineConfiguration->adcVcc = 3.3f;
+}
+
+static void setStepperConfig() {
+	engineConfiguration->idle.stepperDirectionPin = Gpio::A8;
+	engineConfiguration->idle.stepperStepPin = Gpio::C6;
+	engineConfiguration->stepperEnablePin = Gpio::C7;
+}
+
+
+static void setLedPins() {
+	// G12 is error LED, configured in board.mk
+	//engineConfiguration->communicationLedPin = Gpio::G13;
+	//engineConfiguration->runningLedPin = Gpio::G10;
+	//engineConfiguration->warningLedPin = Gpio::G11;
+}
+
+static void setupSdCard() {
+	
+	//SD CARD overwrites
+	engineConfiguration->sdCardSpiDevice = SPI_DEVICE_3;	
+	
+
+	engineConfiguration->is_enabled_spi_3 = true;
+	engineConfiguration->spi3sckPin = Gpio::C10;
+	engineConfiguration->spi3misoPin = Gpio::C11;
+	engineConfiguration->spi3mosiPin = Gpio::C12;
+}
+
+static void setupEGT() {
+	
+	//EGT overwrites
+
+	engineConfiguration->spi2sckPin = Gpio::B13;
+	engineConfiguration->spi2misoPin = Gpio::B14;
+	engineConfiguration->spi2mosiPin = Gpio::B15;
+	engineConfiguration->is_enabled_spi_2 = true;
+
+	engineConfiguration->max31855spiDevice = SPI_DEVICE_2;
+	engineConfiguration->max31855_cs[0] = Gpio::C8;
+	engineConfiguration->max31855_cs[1] = Gpio::C9;
+}
+
+
+void setBoardConfigOverrides() {
+	setupSdCard();
+	setEtbConfig();
+	setStepperConfig();
+	setLedPins();
+	setupEGT();
+
+	engineConfiguration->clt.config.bias_resistor = 2490;
+	engineConfiguration->iat.config.bias_resistor = 2490;
+
+	//SERIAL 
+	engineConfiguration->binarySerialTxPin = Gpio::A9;
+	engineConfiguration->binarySerialRxPin = Gpio::A10;
+	engineConfiguration->tunerStudioSerialSpeed = SERIAL_SPEED;
+	engineConfiguration->uartConsoleSerialSpeed = SERIAL_SPEED;
+
+
+
+	//CAN 1 bus overwrites
+	engineConfiguration->canRxPin = Gpio::D0;
+	engineConfiguration->canTxPin = Gpio::D1;
+
+	//CAN 2 bus overwrites
+	engineConfiguration->can2RxPin = Gpio::B5;
+	engineConfiguration->can2TxPin = Gpio::B6;
+}
+
+static void setupDefaultSensorInputs() {
+
+	engineConfiguration->afr.hwChannel = EFI_ADC_13; //PC3
+	engineConfiguration->afr.hwChannel2 = EFI_ADC_0; //PA0
+	setEgoSensor(ES_14Point7_Free);
+	
+	engineConfiguration->map.sensor.hwChannel = EFI_ADC_2; //PB0
+	engineConfiguration->map.sensor.type = MT_MPXH6400;
+	
+	engineConfiguration->baroSensor.hwChannel = EFI_ADC_NONE;
+
+	engineConfiguration->lps25BaroSensorScl = Gpio::B10;
+	engineConfiguration->lps25BaroSensorSda = Gpio::B11;
+
+}
+
+
+void setBoardDefaultConfiguration(void) {
+	setInjectorPins();
+	setIgnitionPins();
+	setupDefaultSensorInputs();
+	setupVbatt();
```

Comment:
this should probably go in overrides, not default

Reply:
moved vbat and sd card config to overwrites